packages/client/src/__tests__/dataTriggers.test.ts

```diff
@@ -332,4 +333,42 @@ describe("Cloud functions -> Data triggers ->,", () => {
       }
     );
   });
+  describe("createPublicOrgInfo", () => {
+    testWithEmulator(
+      "should update/create general info in organization data to publicOrgInfo collection when organization data is updated",
+      async () => {
+        const { displayName, location, emailFrom } = organization;
+        const orgName = "test-publicOrgInfo";
+
+        const publicOrgPath = `${Collection.PublicOrgInfo}/${orgName}`;
+        const orgPath = `${Collection.Organizations}/${orgName}`;
+
+        // check for non existence of publicOrgInfo before creating a new organization
+        const publicOrgRes = await waitForCondition({
+          documentPath: publicOrgPath,
+          condition: (data) => Boolean(!data),
+        });
+        expect(publicOrgRes).toBeUndefined();
```

Comment:
Right...this check makes sense when you're writing, or debugging the test (during a TDD session), but if you've set up the tests correctly, shouldn't be necessary: The purpose of the test isn't to test whether or not the organization entry is empty.